Lebenslauf.html

```diff
@@ -1,142 +1,152 @@
-  <tr>
-    <td></td>
-    <td>Verkauf von Büchern und anderen Medien, Kundenberatung und -bedienung, Bibliographieren und Recherchieren, Beschaffung von Kundenbestellungen, Sortimentsgestaltung</td>
-  </tr>
-  <tr>
-    <td><i>2010-2012</i></td>
-    <td>Aushilfskraft bei Eismanufaktur Potsdam</td>
-  </tr>
-  <tr>
-    <td></td>
-    <td>Tresen- und Servicekraft</td>
-  </tr>
-  <tr>
-    <td><i>2007-2014</i></td>
-    <td>Hostess & Chefhostess bei Heiden & Wo-Men Veranstaltungsservice</td>
-  </tr>
-  <tr>
-    <td></td>
-    <td>Delegation von Teams mit mehr als fünf Mitarbeitern, Koordination von Veranstaltungsabläufen, Organisation und Nachbereitung vor Ort</td>
-   </tr>
-   </table>
-
-  <h2 style="color:blue;"><h2 style="font-family:verdana;">Ausbildung</h2>
-  <hr>
-
-  <table>
-    <tr>
-      <th>Zeitraum</th>
-      <th>Tätigkeit</th>
-    </tr>
-    <tr>
-      <td>2010-2015</td>
-      <td>B.A. Germanistik und Spanische Philologie, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
-    </tr>
-    <tr>
-      <td>2009-2010</td>
-      <td>Spanische Philologie und Linguistik, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
-    </tr>
-    <tr>
-      <td>2008-2009</td>
-      <td>Französische Philologie und Linguistik, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
-    </tr>
-    <tr>
-      <td>2001-2008</td>
-      <td>Allgemeine Hochschulreife, <a href="https://www.wolkenberg-gymnasium.de/">Wolkenberg Gymnasium Michendorf</a></td>
-     </tr>
-     </table>
-
-  <h3 style="color:green;"><h3 style="font-family:verdana;">Sprachen</h3>
-
-  <table>
-    <tr>
-      <th>Sprache</th>
-      <th>Kenntnisstand</th>
-    </tr>
-    <tr>
-      <td>Deutsch</td>
-      <td>Muttersprache</td>
-    </tr>
-    <tr>
-      <td>Englisch</td>
-      <td>Sehr gute Kenntnisse</td>
-    </tr>
-    <tr>
-      <td>Spanisch</td>
-      <td>Sehr gute Kenntnisse</td>
-    </tr>
-    <tr>
-      <td>Französisch</td>
-      <td>Grundkenntnisse</td>
-    </tr>
-    <tr>
-      <td>Portugiesisch</td>
-      <td>Grundkenntnisse</td>
-    </tr>
-    <tr>
-      <td>Latein</td>
-      <td>kleines Latinum</td>
-     </tr>
-     </table>
-
-  <h3 style="color:green;"><h3 style="font-family:verdana;">Qualifikationen</h3>
-
-  <p>Gitarre (Seit 2005)
-  <br>
-  <p>Chorgesang (Seit 2001)
-
-  <h3 style="color:green;"><h3 style="font-family:verdana;">Kenntnisse</h3>
-
-  <table>
-    <tr>
-      <th>EDV</th>
-      <th>Kenntnisstand</th>
-    </tr>
-   <tr>
-     <td>MS Word 2010</td>
-     <td>Gute Kenntnisse</td>
-   </tr>
-   <tr>
-     <td>MS Excel 2010</td>
-     <td>Grundkenntnisse</td>
-   </tr>
-   <tr>
-     <td>MS Powerpoint 2010</td>
-     <td>Grundkenntnisse</td>
-   </tr>
-   <tr>
-     <td>Markdown</td>
-     <td>Grundkenntnisse</td>
-    </tr>
+
+    <h1 style="font-size:200%; font-family:verdana;">Lebenslauf</h1>
+    <p>Luise Maron</p>
+    <img src="Passfoto.jpg" width="150" height="224">
```

Comment:
You don't need to specify `width` & `height`. You do need to specify an `alt`-attribute tho. The `alt`-attribute is an alternative text, should the image not been displayed. For accessibility reason or due to loading issues.